### PR TITLE
DOCS-4559: Add docs to a new folder and enhance content

### PR DESCRIPTION
**Pull request by anxumalo**
- Created docs folder
- Moved Helm / k8s for AWS instructions from Wiki
- Added Helm / Minikube instructions from helm/a-c-s dir 
- Moved 'Running with Docker Compose', 'Running with Docker Kubernetes' and 'Customising alfresco deployment' sections from README to new docs pages
- Added links to new docs page from README and other cross refs
- Enhanced content
- This update also addresses DOCS-4320 (Helm charts), and DOCS-4319 (reference deployment).

### CODE REVIEW

**Comment by anxumalo** [comment]
@killerboot Update done - please check again.

File: README.md [hunk 1]
```diff
@@ -23,6 +23,10 @@ Alfresco Content Services deployed via docker-compose or Kubernetes contains the
 3. A Postgres DB  
 4. Alfresco Solr6  
 
+## Deployment options
+* [Deploying on AWS with Kubernetes](docs/helm-deploy-aws_cloud.md)
+* [Deploying with Helm on a local host](docs/helm-deployment-localhost.md)
```
Comment (killerboot) [inline README.md]: Is this section describing how to deploy the helm chart on local host with minikube? The name can be confused with the sub section down in README.md called "Minikube" in "Running with Kubernetes".

I think we should split the docker-compose and minikube from README in the same way as "Deploying on AWS with Kubernetes" is done.

Comment (anxumalo) [reply README.md]: Agreed. Will continue working on suggested improvements, as discussed.

File: docs/helm-deployment-aws_cloud.md [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+Adjust the above values accordingly (ex: `--version` of `nginx-ingress`, `controller.service.annotations` etc.)
+
+* To view the AWS ELB DNS name, note down the `nginx-ingress` release from the output of the above command.  It should be something like (Ex: `NAME:  knobby-wolf`)
+```bash
+export INGRESSRELEASE=knobby-wolf
+export ELBADDRESS=$(kubectl get services $INGRESSRELEASE-nginx-ingress-controller --namespace=$DESIREDNAMESPACE -o jsonpath={.status.loadBalancer.ingress[0].hostname})
+echo $ELBADDRESS
+```
+
+* Create an EFS storage on AWS and make sure it's in the same VPC as your cluster by following this [guide](https://github.com/Alfresco/alfresco-dbp-deployment#6-efs-storage-note-only-for-aws).  Make sure you open inbound traffic in the security group to allow NFS traffic.  Below is an example of how to create it with the AWS Cli.
```
Comment (montgolfiere) [inline docs/helm-deployment-aws_cloud.md]: Should we mention S3 Connector (and provide link to docs) as an alternative add-on option for ACS Content Repository ?

File: docs/helm-deployment-minikube.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
+# Alfresco Content Services Deployment with Helm using Minikube
+
+## Prerequisites
+
+To run the Alfresco Content Services (ACS) deployment using Minikube requires:
+
+| Component   | Getting Started Guide |
```
Comment (killerboot) [inline docs/helm-deployment-minikube.md]: This table doesn't look right in web view.

**Pull request merged**